benchopt/benchmark.py

```diff
@@ -449,7 +449,7 @@ def get_all_runs(self, solver_names=None, forced_solvers=None,
 
 
 def _check_name_lists(*name_lists):
-    "Normalize name_list ot a list of lowercase str."
+    "Normalize name_list to a list of lowercase str."
     res = []
     for name_list in name_lists:
         if name_list is None:
```

Comment:
We need to remove the `.lower()` bellow as this causes issue with the python tokens.

Reply:
Well, I feel like it is useful to tolerate both `python-pgd` and `python-PGD`. Another fix would be to support `true` for `True`. See 78ac0cbc863e9d0322f7ac884e74ad5e9fac6332

Reply:
This is already supported as the regex use `IGNORECASE`